### ISSUE TITLE
Mouse selection behavior with overflow: auto

### ISSUE THREAD

**Issue by DaGaMx** [mdn/content#30949]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow

### What specific section or headline is this issue about?

Accessibility concerns

### What information was incorrect, unhelpful, or incomplete?

If you have overflow: hidden and content-editable="true" then the current text cursor isn't used for scrolling when text is selected via the mouse. However, scrolling is done when the keyboard is used.

### What did you expect to see?

It would be nice if the following information could be added (in more formal speech):
If you have a content-editable container with overflow: hidden, the container will not automatically scroll when selecting text.
If you need automatic scrolling, you can use "overflow: auto" and "scrollbar-width: none" instead to enable auto scrolling while selecting text with the mouse.

### Do you have any supporting links, references, or citations?

https://bugzilla.mozilla.org/show_bug.cgi?id=903307
https://stackoverflow.com/questions/16670931/hide-scroll-bar-but-while-still-being-able-to-scroll/49278385#49278385

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/overflow`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/overflow/index.md
* Last commit: https://github.com/mdn/content/commit/b8935e8b2b5fc2315258104266ce2b05a0631802
* Document last modified: 2023-08-16T12:58:14.000Z

</details>

**Comment by OnkarRuikar** [comment]
> ## What did you expect to see?
> It would be nice if the following information could be added (in more formal speech):
> If you have a content-editable container with overflow: hidden, the container will not automatically scroll when selecting text.

We can not suggest developers to use `overflow: hidden` or `scrollbar-width: none` for scrollable content because [it is bad for accessibility](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width#accessibility_concerns). Also, at the moment `scrollbar-width` is [not supported by all the browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width#browser_compatibility).

**Comment by estelle** [comment]
This is against best practices, so should not document it. Closing.

**Comment by DaGaMx** [comment]
Just to be entire here:
In my case I've got a single line <div contenteditable="true">, which is used as replacement for a <input type="text"> and should look similar (<input type="text"> has no scrollbars).
I'm only using the div, because I can do syntax highlighting there.
Without scrollbar-width: none (or the scrollbar-color: transparent for Firefox) this is hardly to implement for a single line control.

The target for my web application are only Desktop computers.
However, if I switch my web browser to the mobile view I am able to scroll the text in the div like in a text input field, but the page does move a little bit, too.

While reading the MDN article about content-editable, I wasn't able to find any useful information why the keyboard was able to scroll, but not the mouse, when you hid the scollbars. So, I made this request here.
